cities/berlin.json

```diff
@@ -2147,249 +2163,243 @@
                 "opening_hours_unclassified": null
             },
             "type": "Feature"
         },
         {
             "geometry": {
                 "coordinates": [
-                    13.52667,
-                    52.45864
+                    13.35445,
+                    52.48779
                 ],
                 "type": "Point"
             },
             "properties": {
-                "title": "Wochenmarkt Berlin-Oberschöneweide, Rathenauplatz, 12459 Berlin",
-                "location": "Wochenmarkt Berlin-Oberschöneweide, Rathenauplatz, 12459 Berlin",
-                "opening_hours": "We 09:00-16:00",
+                "title": "Ökomarkt an der Akazienstraße",
+                "location": "Ökomarkt an der Akazienstraße",
+                "opening_hours": "Th 12:00-18:00",
                 "opening_hours_unclassified": null
             },
             "type": "Feature"
         },
         {
             "geometry": {
                 "coordinates": [
-                    13.55012,
-                    52.43866
+                    13.33986,
+                    52.52606
                 ],
                 "type": "Point"
             },
             "properties": {
-                "title": "Wochenmarkt Adlershof, Dörpfeldstraße, 12489 Berlin",
-                "location": "Wochenmarkt Adlershof, Dörpfeldstraße, 12489 Berlin",
-                "opening_hours": "We,Th 08:00-16:00",
-                "opening_hours_unclassified": null
+                "title": "Ökomarkt an der Thusnelda-Allee, Thusnelda-Allee 1, 10555 Berlin",
+                "location": "Ökomarkt an der Thusnelda-Allee, Thusnelda-Allee 1, 10555 Berlin",
+                "opening_hours": null,
+                "opening_hours_unclassified": "Mi 12:00 - 18:00 (Winter) 12:00 - 19:00 (Sommer)"
             },
             "type": "Feature"
         },
         {
             "geometry": {
                 "coordinates": [
-                    13.51276,
-                    52.45566
+                    13.34409,
+                    52.51714
                 ],
                 "type": "Point"
             },
             "properties": {
-                "title": "Wochenmarkt Schellerstraße Ecke Spreestraße",
-                "location": "Wochenmarkt Schellerstraße Ecke Spreestraße",
-                "opening_hours": "Th 08:00-16:00",
+                "title": "Ökomarkt im Hansaviertel, Parkplatz vor der St.-Ansgar-Kirche, Altonaer Straße",
+                "location": "Ökomarkt im Hansaviertel, Parkplatz vor der St.-Ansgar-Kirche, Altonaer Straße",
+                "opening_hours": "Fr 12:00-18:30",
                 "opening_hours_unclassified": null
             },
             "type": "Feature"
         }
     ],
-    "metadata": {
-        "data_source": {
-            "title": "Stadt Berlin, CC BY 3.0 DE, aktualisiert am 03.05.2019",
-            "url": "https://daten.berlin.de/datensaetze/wochen-und-tr%C3%B6delm%C3%A4rkte-0"
-        }
-    },
```

Comment:
The `metadata` block cannot be removed. It is still added manually, see this [`README.md`](https://github.com/wo-ist-markt/wo-ist-markt.github.io/blob/master/preprocessing/README.md).